main.js

```diff
@@ -0,0 +1,28 @@
+/* global data */
+/* exported data */
+var $car = document.querySelector('#car');
+
+document.addEventListener('keydown', function (event) {
+  data.previousView = data.view;
+  if (event.key === 'ArrowUp') {
+    $car.setAttribute('data-view', 'up');
+    data.view = 'up';
+    $car.className = 'up';
+    $car.style = 'transform: rotate(-90deg)';
```

Comment:
You should make a class for these changes to the `transform` property and just add that class to the element

Reply:
![driving-game-turn-car-css](https://user-images.githubusercontent.com/48267398/155636758-08c8539e-fbeb-4f04-b7fe-eeae2a1fab47.png)